content/events/one-pager.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Modern Infrastructure as Code
+type: page
+layout: event-one-pager
+url: /show
```

Comment:
Should this have a more specific URL, like /events/[date] or the like?

We've also talked about creating an Events listing, so perhaps driving this these with Hugo data somehow might better enable figure events to slot in more automatically?

Reply:
I believe that we're attributing traffic and reporting based on utm codes used in campaigns.

Reply:
The URL is general because this specific page is going to linked to from a QR code on our printed banner and slide deck. So we want to maintain the ability to change the page content without having to reprint the QR code.

Reply:
Ah, okay -- that makes sense. Thanks for the clarification.

Reply:
It does seem like maybe this URL could be slightly less detached from anything concrete, though. Maybe I'm overthinking it, but I can imagine, say, having two banners or something -- do you want to know which one was interacted with? 

If not, just tell me to shut up. It's just bugging me but again, I'm probably overthinking it.

Reply:
It's also sort of bugging me that the name is /events/one-pager, and the layout is /page/event-one-pager, but the url is /show. Are there going to be "event-two-pager"s? The naming just isn't lining up for me.

Like, if everything were named "event" -- /event, /event.md, /event.html -- that'd make sense. Or even /show (although event seems much clearer). Do you see what I mean? I'm just not understanding why these names are what they are.

Reply:
> Like, if everything were named "event" -- /event, /event.md, /event.html -- that'd make sense. Or even /show (although event seems much clearer). Do you see what I mean? I'm just not understanding why these names are what they are.

I agree.